frontend/doc-site/05__Annotate and Analyze Your Data/5_4__Annotating Data.mdx

```diff
@@ -84,7 +84,7 @@ cellxgene launch mydata.h5ad --annotations-dir path/to/annotations-directory/
 
 For each user, annotations will be saved as follows:
 Each user will be prompted to enter a name for their annotations the first time they create a new category
-We also assign a unique identifier in the form of an 8-character suffix, `########`; this helps cellxgene identify their specific file to avoid overwriting others' work
+We also assign a unique identifier in the form of an 8-character suffix, `########`; this helps CELLxGENE identify their specific file to avoid overwriting others' work
```

Comment:
```suggestion
We also assign a unique identifier in the form of an 8-character suffix, `########`; this helps CELLxGENE Annotate identify their specific file to avoid overwriting others' work
```